substrate/frame/balances/src/tests/fungible_tests.rs

```diff
@@ -653,6 +653,7 @@ fn lone_hold_consideration_works() {
 			assert_eq!(Balances::balance_on_hold(&TestId::Foo, &who), 10);
 
 			assert_ok!(Balances::hold(&TestId::Foo, &who, 5));
+			assert_eq!(events(), [RuntimeEvent::Balances(crate::Event::Held { reason: TestId::Foo, who: who.clone(), amount: 10 }), RuntimeEvent::Balances(crate::Event::Held { reason: TestId::Foo, who: who.clone(), amount: 5 })]);
 			assert_eq!(Balances::balance_on_hold(&TestId::Foo, &who), 15);
```

Comment:
maybe we can also have a test for assert `Released` event?

Reply:
We can add this to `unbalanced_trait_set_balance_works()` in `fungible_test.rs`:

```rust
....
assert_ok!(<Balances as fungible::MutateHold<_>>::release(&TestId::Foo, &1337, 60, Exact));
System::assert_last_event(RuntimeEvent::Balances(crate::Event::Released { reason: TestId::Foo, who: 1337, amount: 60 }));
....
```

Reply:
@Yung-Beef can you please do this.